.travis/complie_check.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e 
+dist_list=$(go tool dist list)
+
+for dist in ${dist_list}; do
+    GOOS=$(echo ${dist} | cut  -d "/" -f 1)
+    GOARCH=$(echo ${dist} | cut -d "/" -f 2)
+    set +e
+    echo "Checking compile support for ${GOOS}/${GOARCH}"
```

Comment:
This seems unnecessary. We already get 1 line of output for every GOOS/GOARCH pair. Either that it is skipped or that we're attempting a build.

Reply:
This has been updated to print one statement per platform.